src/text-fragment-utils.js

```diff
@@ -482,8 +482,8 @@ export const markRange = (range, documentToProcess = document) => {
         acceptNode: function(node) {
           if (!range.intersectsNode(node)) return NodeFilter.FILTER_REJECT;
 
-          if (BLOCK_ELEMENTS.includes(node.tagName.toUpperCase()) ||
-              node.nodeType === Node.TEXT_NODE)
+          if (node.nodeType === Node.TEXT_NODE ||
+              BLOCK_ELEMENTS.includes(node.tagName.toUpperCase()))
```

Comment:
`node.tagName` is undefined (at least in WebKit) when `nodeType === Node.TEXT_NODE`, so we check `nodeType` first